.circleci/config.yml

```diff
@@ -39,7 +39,18 @@ jobs:
       - run:
           name: Deploy demo
           command: |
-            gh-pages --dist demo -m "Update demo [ci skip]"
```

Comment:
@stephanebisson not sure why this line is changed...
is this correct?

Reply:
We need the original version with the custom message.

Reply:
fixed